arbitrum-docs/launch-orbit-chain/how-tos/arbos-upgrade.md

```diff
@@ -16,17 +16,23 @@ import PublicPreviewBannerPartial from '../../node-running/partials/_upgrade-cad
 
 #### Step 1: Update Nitro on nodes and validators
 
-Refer to the release notes supporting the ArbOS version that you're upgrading to. You'll find a specific version of the Nitro software stack specified. For example, if your upgrade targets ArbOS 11, you'd use Nitro `v2.2.2` (Docker image: `offchainlabs/nitro-node:v2.2.2-8f33fea`) or higher. This is the version of the Nitro stack that needs to be running on each of your Orbit chain's nodes.
+Refer to the release notes supporting the ArbOS version that you're upgrading to. You'll find a specific version of the Nitro software stack specified. For example, if your upgrade targets ArbOS 11, you'd use Nitro `v2.3.0` (Docker image: `offchainlabs/nitro-node:v2.3.0-3e14543`) or higher. This is the version of the Nitro stack that needs to be running on each of your Orbit chain's nodes.
```

Comment:
I think this should say ArbOS 20

Reply:
can we also link to https://github.com/OffchainLabs/nitro/releases